src/pages/Lend/FilterAlpha/LendFilterAlpha.vue

```diff
@@ -11,6 +11,41 @@
 			</div>
 			<div class="lg:tw-absolute lg:tw-w-full lg:tw-h-full lg:tw-top-0 lg:tw-pt-8">
 			</div>
+			<kv-page-container>
+				<h2 class="tw-mb-2">
+					Make a loan, change a life
+				</h2>
+				<h3 class="tw-mb-3">
+					Each Kiva loan helps people build a better future for themselves and their families.
+				</h3>
+				<kv-grid class="tw-grid-cols-2 md:tw-grid-cols-3">
+					<div class="tw-bg-gray-300 text-center sm:text-left"><p>Filters</p>
+					</div>
+
+					<div class="tw-col-span-2 tw-bg-white-300">
+						<div class="tw-bg-gray-300 tw-h-30">Search Loans</div>
+						<p>Loan Count</p>
+                        <kv-grid class="tw-grid-rows-4">
+							<div class="tw-bg-gray-300 tw-h-30">Loan Card 1</div>
```

Comment:
We use a customized spacing system at Kiva, which isn't quite like the docs in Tailwind. That's why these `tw-h-30` classes aren't making things taller right now.

You can either remove these classes, or you could use `tw-h-4` which is 32px in our system, so almost 30 :)

If you're using VsCode, I'd recommend adding the [Tailwind intellisense plugin](https://github.com/tailwindlabs/tailwindcss-intellisense). It will auto-complete classes for you so you can tell which heights are available in Kiva's system.